### PR TITLE
added house - Android tool

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1407 .

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit af2c876324c37e7162b95d8a3a4aa54bad1a53ba: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        4 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x05b-Basic-Security_Testing.md [hunk 1]
```diff
@@ -684,6 +684,14 @@ public void android.view.View.setTooltipText(java.lang.CharSequence)
 
 In the end, it is up to you to decide where would you like to work with the data. Sometimes it will be more convenient to do it from JavaScript and in other cases Python will be the best choice. Of course you can also send messages from Python to JavaScript by using `script.post`. Refer to the Frida docs for more information about [sending](https://www.frida.re/docs/messages/#sending-messages-from-a-target-process "Sending messages from a target process") and [receiving](https://www.frida.re/docs/messages/#receiving-messages-in-a-target-process "Receiving messages in a target process") messages.
 
+##### House
+
+[House](https://github.com/nccgroup/house "House") is a runtime mobile application analysis toolkit for Android Apps, developed and maintained by NCC Group and is written in Python.
+
+It's leveraging on a running Frida Server on a rooted device or the Frida Gadget in a repackaged Android App. The intention of House is to allow an easy way for prototyping of Frida scripts by offering a Web GUI.
```
Comment (cpholguera) [inline Document/0x05b-Basic-Security_Testing.md]: Consistency: server/gadget/app with the rest of the guide + little fixes
```suggestion
It's leveraging a running Frida server on a rooted device or the Frida gadget in a repackaged Android app. The intention of House is to allow an easy way of prototyping Frida scripts via its convenient web GUI.
```

File: Document/0x05b-Basic-Security_Testing.md [hunk 2]
```diff
@@ -684,6 +684,14 @@ public void android.view.View.setTooltipText(java.lang.CharSequence)
 
 In the end, it is up to you to decide where would you like to work with the data. Sometimes it will be more convenient to do it from JavaScript and in other cases Python will be the best choice. Of course you can also send messages from Python to JavaScript by using `script.post`. Refer to the Frida docs for more information about [sending](https://www.frida.re/docs/messages/#sending-messages-from-a-target-process "Sending messages from a target process") and [receiving](https://www.frida.re/docs/messages/#receiving-messages-in-a-target-process "Receiving messages in a target process") messages.
 
+##### House
+
+[House](https://github.com/nccgroup/house "House") is a runtime mobile application analysis toolkit for Android Apps, developed and maintained by NCC Group and is written in Python.
```
Comment (cpholguera) [inline Document/0x05b-Basic-Security_Testing.md]:
```suggestion
[House](https://github.com/nccgroup/house "House") is a runtime mobile application analysis toolkit for Android apps, developed and maintained by the NCC Group and is written in Python.
```

**Comment by sushi2k** [comment]
Thanks @cpholguera. Just merged your changes. Please approve 😊

**Comment by OMTGreleaser** [comment]
Results for commit cf460c8e2369dd8617c9d6feebde63013efab34b: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        4 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**